Makefile

```diff
@@ -22,22 +20,18 @@ clean:
 tools: check-version
 	@echo "=== $(INTEGRATION) === [ tools ]: Installing tools required by the project..."
 	@go get $(GOTOOLS)
-	@gometalinter.v2 --install
+	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.33.0
 
 tools-update: check-version
 	@echo "=== $(INTEGRATION) === [ tools-update ]: Updating tools required by the project..."
 	@go get -u $(GOTOOLS)
-	@gometalinter.v2 --install
+	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.33.0
 
-deps: tools deps-only
-
-deps-only:
-	@echo "=== $(INTEGRATION) === [ deps ]: Installing package dependencies required by the project..."
-	@govendor sync
+deps: tools
 
 validate: deps
 	@echo "=== $(INTEGRATION) === [ validate ]: Validating source code running gometalinter..."
-	#@gometalinter.v2 --config=.gometalinter.json ./...
+	@./bin/golangci-lint run
```

Comment:
not sure if the final approach is the one on nri-nginx @davidbrota  is working on , but i would go for that or just de-scope this for now.

Reply:
Yes, I will modify it to the new version so it's already done

Reply:
My initial plan was to add the new golang-ci tool version together with the semgrep in all our repos in another PR once I have all the staticAnalysis stuff done, do you agree to leave it for that PR?